kvdb-rocksdb/src/lib.rs

```diff
@@ -268,9 +269,8 @@ fn is_corrupted(err: &Error) -> bool {
 /// Generate the options for RocksDB, based on the given `DatabaseConfig`.
 fn generate_options(config: &DatabaseConfig) -> Options {
 	let mut opts = Options::default();
-	let columns = config.columns.unwrap_or(0);
 
-	if columns == 0 {
+	if config.columns.get() == 1 {
```

Comment:
please remove this section as it was only required for setting up the default column. Now that we only use column families, this is implied by [this option](https://github.com/paritytech/parity-common/blob/2c26418244653e4a0134f1dfaf5a6079fa5589cf/kvdb-rocksdb/src/lib.rs#L203).